### DISCUSSION TITLE
Require 0-point achievements to have been earned in order to show up in "latest masters" list.

### DISCUSSION THREAD

**Discussion by redwizard42** [RetroAchievements/RAWeb#1349] [Ideas]
Currently if you have earned every achievement except one or more 0-point achievements, you'll show up in the latest masters list. This is probably actually a bug, but I wasn't sure, so here we are.  This especially looks weird for DevQuest 004 and DevQuest 015.

**Comment by redwizard42** [comment]
Can this check "date of latest achievement earned from all core achievements, must have core achievements earned = total core achievements" for appearing in latest masters?

**Comment by Jamiras** [comment]
Mastery is determined as "having achievements earned = achievements published".

Games with 0 points now show the number of unlocked achievements in the "Latest Masters/High Scores" list. (#2583).
